example/spec/parallel/testAPI/bulkOperationsSuccessSpec.js

```diff
@@ -273,10 +275,19 @@ describe('POST /granules/bulk', () => {
         } catch (error) {
           throw new SyntaxError(`getAsyncOperationBody.output is not valid JSON: ${getAsyncOperationBody.output}`);
         }
+
+        await getGranuleWithStatus({
+          prefix,
+          granuleId: JSON.parse(getAsyncOperationBody.output)[0],
+          status: 'running',
+          timeout: 120,
+          updatedAt: ingestedGranule.updatedAt,
+        });
         expect(output).toEqual([granuleId]);
       }
     });
 
+    // xit comment: https://github.com/nasa/cumulus/pull/2146/files
```

Comment:
Added an explicit reference to the PR in the comments for future ease of reference